Source/WTF/wtf/DateMath.cpp

```diff
@@ -884,14 +884,14 @@ double parseDateFromNullTerminatedCharacters(const char* dateString, bool& isLoc
 
             skipSpacesAndComments(dateString);
 
-            if (startsWithLettersIgnoringASCIICase(dateString, "am"_s)) {
+            if (startsWithLettersIgnoringASCIICase(StringView { dateString }, "am"_s)) {
```

Comment:
Note that here, mishandling ASCII might not matter so much. And maybe we could probably make a more efficient version if we overloaded for `const char*` instead of casting to `StringView`. Not sure at all it’s important.

Reply:
This comment makes more sense if you read the other comments below. I didn’t realize this was "higher" in the file.